packages/arb-token-bridge-ui/src/hooks/__tests__/useNetworks.test.ts

```diff
@@ -33,85 +33,195 @@ describe('sanitizeQueryParams', () => {
     localStorageGetItemMock.mockReset()
   })
 
-  it('sets the default values for `sourceChainId` and `destinationChainId` when both `sourceChainId` and `destinationChainId` are `undefined`', () => {
-    const result = sanitizeQueryParams({
-      sourceChainId: undefined,
-      destinationChainId: undefined
-    })
-    expect(result).toEqual({
-      sourceChainId: ChainId.Ethereum,
-      destinationChainId: ChainId.ArbitrumOne
-    })
-  })
+  describe('when `destinationChainId` is valid', () => {
+    it('and `sourceChainId` is valid should not do anything', () => {
```

Comment:
so right now i can read this line and understand it properly because I've been looking at this PR

but i worry down the line this doesn't have a very high readability

plus, both source chain id and destination chain id should be equally important conditions, so i would do this instead:

```typescript
describe('when `destinationChainId` is valid and `sourceChainId` is valid', () => {
    it('should not do anything', () => {}
}
describe('when `destinationChainId` is valid and `sourceChainId` is invalid', () => {
    it('should set `sourceChainId`', () => {}
}
...
```

There are 2 vars with 3 possible values each (undefined/valid/invalid) so there should be 3x3 = 9 cases here
i see them all in place in this file but they should have parallel status given that the 2 vars' **initial** values have no dependency on each other